### ISSUE TITLE
Add Contributing Guide

### ISSUE THREAD

**Issue by chrismclarke** [ONEARMY/academy#78]
Had a couple message from people hoping to contribute, so it would be good to add some clearer guidance like we do on the community platform. Ideally it should cover:

- How to fork the repo (basic github stuff, probably taken from community platform)
- How to get platform working locally (mostly just docusaurus docs)
- Basic file/folder structure
- Making edits (any suggested best ways to go about it)
- Making prs
- Who to reach out to if further questions

**Comment by darigovresearch** [comment]
@chrismclarke we're happy to take a look at this if this is still considered a worthwhile improvement, is that the case?

**Comment by chrismclarke** [comment]
Hi @darigovresearch , thanks for offering to help on this. I would say definitely really useful to have the basic info covered, although let me ask @davehakkens his thoughts regarding people making suggestions/additions to docs - is it something we want to be encouraging? Is there a best way that people could go about it if the do want to?

**Comment by davehakkens** [comment]
Yes sounds usefull! We are currently setting up an "precious plastic maintaince team" they would keep an overview of all the contributions and edits from the community to make sure it doesn't become chaos. Like a moderater group. But either way we would need to outline this contribution guide.

**Comment by darigovresearch** [comment]
@davehakkens @chrismclarke We would be happy to take part in a "maintenance team" as we've been building the machines since v2 of the machines & if you think it would be of help (see some of the small fixes we've already made in some of the kits). 

Is there any progress on the outline of this contribution guide? 

Happy to provide a starting point to adapt if this helps move the issue along!
